tests/mentorship_relation/test_api_accept_request.py

```diff
@@ -3,13 +3,14 @@
 from datetime import datetime, timedelta
 
 from app import messages
+from app.database.models.mentorship_relation import MentorshipRelationModel
 from app.database.models.tasks_list import TasksListModel
 from app.database.sqlalchemy_extension import db
-from app.database.models.mentorship_relation import MentorshipRelationModel
 from app.utils.enum_utils import MentorshipRelationState
 from tests.mentorship_relation.relation_base_setup import MentorshipRelationBaseTestCase
 from tests.test_utils import get_test_request_header
 
+
```

Comment:
<img width="690" alt="pep8" src="https://user-images.githubusercontent.com/40357511/71448997-ec10ba00-2744-11ea-805c-c9b508717634.png">

PyCharm says it's against PEP8

Reply:
Shall I ignore it?

Reply:
@bartekpacia Actually in open source it is considered a good practice to solve a single issue at a time.Some other students have also raised the same doubt and I have asked them to do the same.We might create new tasks for resolving pep8 warnings in files and then this can be taken up.I really appreciate the efforts that you have put in for the same but for now ignoring it would be the best :)

Reply:
Okay, thanks for fast reply:)